### PR TITLE
[SOAR-284] - Block Sender With Teams (Try 2)

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Description 

This PR adds the Block Sender With Teams workflow 

## Validation: 
```
rmt-mbp-5357:Block_Sender_With_Teams jmcadams$ icon-validate .
[*] Validating workflow at .

[*] Running Integration Validators...
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 24.011000000000003ms
```

## Workflow: 
workflow1

## Teams Flow: 
teams_flow1

## Artifacts: 
image

### CODE REVIEW

**Comment by jmcadams-r7** [comment]
Also see: 
https://github.com/rapid7/insightconnect-workflows/pull/79

File: workflows/Block_Sender_in_Office_365_with_Teams/help.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+# Key Features
+
+* A simple command sent from Microsoft Teams will block a malicious email or domain using Exchange transport rules
+
+# Requirements
+
+The following connections will need to be setup: 
+
+* Microsoft Teams
```
Comment (jschipp-r7) [inline workflows/Block_Sender_in_Office_365_with_Teams/help.md:13]: Linke to the newly published doc

Comment (jschipp-r7) [inline workflows/Block_Sender_in_Office_365_with_Teams/help.md:13]:
```suggestion
* [Microsoft Teams](https://insightconnect.help.rapid7.com/docs/microsoft-teams)
```

File: workflows/Block_Sender_in_Office_365_with_Teams/help.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+* 1.0.0 - Initial workflow
+
+# Links
+
+## References
+
+* [Microsoft Office365](https://www.office.com)
+* [Microsoft Teams](https://teams.microsoft.com)
+* [Mail Flow Rules (Transport Rules)](https://docs.microsoft.com/en-us/exchange/security-and-compliance/mail-flow-rules/mail-flow-rules)
```
Comment (jschipp-r7) [inline workflows/Block_Sender_in_Office_365_with_Teams/help.md:59]:
```suggestion
* [Mail Flow Rules (Transport Rules)](https://docs.microsoft.com/en-us/exchange/security-and-compliance/mail-flow-rules/mail-flow-rules)
* [Microsoft Teams Plugin Configuration](https://insightconnect.help.rapid7.com/docs/microsoft-teams)
* [Microsoft Office365 Email Security Plugin Configuration](https://insightconnect.help.rapid7.com/docs/mass-delete-with-powershell#section-set-up-office-365-dependencies)
```

**Pull request merged**